tensornetwork/backends/shell/shell_backend.py

```diff
@@ -19,6 +19,9 @@
 import functools
 import operator
 from tensornetwork.backends import base_backend
+from ..numpy import numpy_backend

```

Comment:
Doesn't doing this defeat the purpose? We are now importing all of the libraries when using Shell.

Reply:
I'm unclear what the shell backend is for. If there is documentation, I haven't found it yet. But in the comments about dtypes the code states "# shell supports everything". So yes, you have to import everything to create the list of acceptable date types when you use the shell backend.

Reply:
Shell backend is a future feature. It's being used to help keep track of the real shape of a tensor for our jax_tpu backend (that is still in development) and was planned to be used to help calculate contraction paths before we integrated with opt_einsum